src/runtime/runtime.js

```diff
@@ -440,16 +440,6 @@ export class Runtime {
       return runtime.getPropensityModule();
     });
   }
-
```

Comment:
I'm removing this in a different PR.  Just FYI.  I don't really care if you do also.

Reply:
Ack. I'll remove it as well.

Reply:
I'll remove it as well.